.golangci-kal.yml

```diff
@@ -0,0 +1,76 @@
+version: "2"
+run:
+  go: "1.24"
```

Comment:
Can we avoid this configuration to get version from go.mod automatically?

Reply:
I didn't need this file.